IoTuring/Entity/Deployments/Fanspeed/Fanspeed.py

```diff
@@ -0,0 +1,279 @@
+import psutil
+
+from IoTuring.Entity.Entity import Entity
+from IoTuring.Entity.EntityData import EntitySensor
+from IoTuring.Entity.ValueFormat import ValueFormatterOptions
+from IoTuring.MyApp.SystemConsts import OperatingSystemDetection as OsD
+from IoTuring.Configurator.MenuPreset import MenuPreset
+
+
+supports_win_fanspeed = False
+supports_linux_fanspeed = True
+supports_macos_fanspeed = True
+
+KEY_FANSPEED = "fanspeed"
+KEY_FANLABEL = "fanlabel"
+
+FALLBACK_PACKAGE_LABEL = "controller"
+FALLBACK_SENSOR_LABEL = "fan"
+
+CONFIG_KEY_CONTROLLER = "controller"
+CONFIG_KEY_THRESHOLD = "threshold"
+
+FANSPEED_DECIMALS = 0
+
+
+class Fanspeed(Entity):
+    """Entity to read fanspeed"""
+
+    NAME = "Fanspeed"
+
+    def Initialize(self) -> None:
+        """Initialize the Class, setup Formatter, determin specificInitialize and specificUpdate depending on OS"""
+        self.fanspeedFormatOptions = ValueFormatterOptions(
+            value_type=ValueFormatterOptions.TYPE_ROTATION, decimals=FANSPEED_DECIMALS
+        )
+
+        self.specificInitialize = None
+        self.specificUpdate = None
+
+        if OsD.IsLinux() or OsD.IsMacos():
+            if not hasattr(psutil, "sensors_fans"):
```

Comment:
```suggestion
            if not bool(psutil.sensors_fans()):
```
`psutil.sensors_fans()` returns an empty dict if there are no fans. Tested on my rpi

Reply:
But that attribute may also be not present in the psutil package, like in my setup.

Reply:
According to documentation Mac is not supported: https://psutil.readthedocs.io/en/latest/#psutil.sensors_fans

And in this example: https://github.com/giampaolo/psutil/blob/master/scripts/fans.py 
No attribute means not supported (like a mac), empty dict means supported, but no fans (like an rpi)